### PR TITLE
Markshake patch 1

### PR DESCRIPTION
**Pull request by markshake**
This is new PR for PISO code.

### CODE REVIEW

**Comment by yungyuc** [comment]
I will decline this PR because (i) you are aware of my latest update at master branch and (ii) your ascii files are still dos (crlf) format, not Unix (lr).

File: code/Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Default Target
+.PHONY: all
+all: CAVITY_2D_STAGGERED
+
+# CAVITY_2D_STAGGERED
+CAVITY_2D_STAGGERED: clean
+	gfortran -o CAVITY_2D_STAGGERED CAVITY_2D_STAGGERED.f90
```
Comment (yungyuc) [inline code/Makefile:7]: The file in directory ends with “F”90, but you name it “f”90 here.  It won’t work under a case sensitive file system like what Linux uses.

**Comment by yungyuc** [comment]
Please create a new PR by branching from the latest master and fixing the Makefile mistake.  If you think it’s easier to just update this PR in place, feel free to reopen.

**Comment by markshake** [comment]
Sorry,  I didn't notice that when testing the linux system, I changed the program's extension.

**Comment by yungyuc** [comment]
No problem.  I saw your new PR #6 and let’s work on that.

**Pull request closed without merging**